package.json

```diff
@@ -48,6 +50,7 @@
     "@types/humanize-duration": "^3.25.0",
     "@types/torrent-search-api": "^2.0.0",
     "@types/uuid": "^8.3.0",
+    "esbuild-node-tsc": "^1.6.1",
```

Comment:
What's the benefit over `esbuild --bundle --platform node --outdir dist`?

Reply:
takes the configurations from `tsconfig.json`